depin/services/roles/ethereum/tasks/commands/install.yml

```diff
@@ -0,0 +1,162 @@
+---
+- name: Install dependencies
+  become: true
+  ansible.builtin.apt:
+    name:
+      - openjdk-21-jdk
```

Comment:
Will probably be dependent on what clients have been selected right?

Reply:
Nimbus is a new ethereum consensus client with a different configuration for start.